handlers/create.go

```diff
@@ -26,6 +27,7 @@ const (
 )
 
 var RegExDockerMsg = regexp.MustCompile("msg=.*")
+var RegExHyphen = regexp.MustCompile(`([a-z])([A-Z])`)
```

Comment:
Is there an explicit reason for using backtick in this regex and double quote in the one above it?
Either is fine, but consistency would be nice